Resources/doc/cookbook/extraction_configs.rst

```diff
@@ -10,8 +10,8 @@ also set-up some pre-defined settings via the configuration:
     jms_translation:
         configs:
             app:
-                dirs: [%kernel.root_dir%, %kernel.root_dir%/../src]
-                output_dir: %kernel.root_dir%/Resources/translations
+                dirs: ["%kernel.root_dir%", "%kernel.root_dir%/../src]"

```

Comment:
You are correct. That was a typo. It is fixed in 22e97a8f0b6a39d832a187b8704c727216de897b